ambry-router/src/test/java/com/github/ambry/router/RouterTestHelpers.java

```diff
@@ -132,8 +132,7 @@ static void testWithErrorCodes(ServerErrorCode[] serverErrorCodesInOrder, Partit
    * @param errorCodeChecker Performs the checks that ensure that {@link RouterErrorCode#TooManyRequests} is returned .
    * @throws Exception
    */
-  static void testWithQuotaRejection(ErrorCodeChecker errorCodeChecker)
-      throws Exception {
+  static void testWithQuotaRejection(ErrorCodeChecker errorCodeChecker) throws Exception {
```

Comment:
rm formatting edits. 
PR can be smaller if all formatting edits are moved to a different PR.